include/silo/database.h

```diff
@@ -70,9 +83,23 @@ class Database {
 
    [[nodiscard]] const PangoLineageAliasLookup& getAliasKey() const;
 
+   const silo::storage::column::DateColumn& getDateColumn(std::string name) const;
```

Comment:
```suggestion
   const silo::storage::column::DateColumn& getDateColumn(const std::string& name) const;
```
According to clang tidy, this saves some copies.